src/frontend/packages/lib_classroom/src/components/DashboardClassroomAskUsername/index.spec.tsx

```diff
@@ -31,6 +31,9 @@ describe('<DashboardClassroomAskUsername />', () => {
     fireEvent.change(inputUsername, { target: { value: 'Joe' } });
     expect(mockSetUserFullname).toHaveBeenCalledWith('Joe');
 
+    fireEvent.submit(inputUsername);
```

Comment:
as you are adding a form, it is not possible to submit on this form ? I don't remember if you use `screen.getByRole('form')`